ports/libtorch/portfile.cmake

```
@@ -0,0 +1,189 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO pytorch/pytorch
+    REF v1.10.0
+    SHA512 92b70e6170a7f173c4a9cb29f6cec6dfa598587aa9cf6a620ec861b95da6ea555cbc7285914c0dab6cfc8af320fad4999be4a788acc1f15140664a67ad9dc35d
+    HEAD_REF master
+    PATCHES
+        fix-cmake.patch
+        fix-sources.patch
+)
+file(REMOVE_RECURSE "${SOURCE_PATH}/caffe2/core/macros.h") # We must use generated header files
+
+find_package(Python3 REQUIRED COMPONENTS Interpreter)
+
+# run `pip install ${package}`
+function(vcpkg_pip_install)
```

```suggestion
function(pip_install)
```

we (the vcpkg team) reserve any names starting with `vcpkg`, so we need to rename this.